analyzers/src/SonarAnalyzer.CSharp/Metrics/CSharpMetrics.cs

```diff
@@ -34,7 +34,11 @@ public CSharpMetrics(SyntaxTree tree, SemanticModel semanticModel) : base(tree)
                 throw new ArgumentException(InitializationErrorTextPattern, nameof(tree));
             }
 
-            lazyExecutableLines = new Lazy<ImmutableArray<int>>(() => CSharpExecutableLinesMetric.GetLineNumbers(tree, semanticModel));
+            lazyExecutableLines = new Lazy<ImmutableArray<int>>(() =>
+                // ToDo: this will be implemented by https://github.com/SonarSource/sonar-dotnet/pull/7915
```

Comment:
We need this until the referenced PR is merged. Otherwise we will report invalid lines and the end step will fail.